server/protocols/jmap-integration-testing/memory-jmap-integration-testing/pom.xml

```diff
@@ -177,18 +177,4 @@
         </dependency>
     </dependencies>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <reuseForks>false</reuseForks>
-                    <forkCount>1C</forkCount>
-                    <excludedGroups combine.self="override">org.apache.james.jmap.categories.CassandraAndElasticSearchOnly</excludedGroups>
```

Comment:
I guess there are here for a good reason: CassandraAndElasticSearch only are for tests that does not work with a basic in memory implementation, like for example stemming.

Reply:
I think what you says don't make sense as CassandraAndElasticSearch is a subset of tests that we want to execute against cassandra+es and not a subset of test designed for Cassandra+ES

Won't fight on this - I just want my build time down to a reasonable amount.

Reply:
why is it named CassandraAndElasticSearchOnly then?

Reply:
oh strange, it's not used at all?

Reply:
For running the subset that we want to run cassandra +es on